roles/puddle/tasks/distill/configure.yml

```diff
@@ -1,24 +1,19 @@
 ---
 - name: add ceph distill config dir
   file:
-    path: /etc/distill/ceph
+    path: /etc/distill/
     owner: root
     group: root
     mode: 0755
     state: directory
 
-- name: add distill config files
-  template:
-    src: "../../templates/distill/{{ item }}"
-    dest: "/etc/distill/ceph/{{ item }}"
-    owner: root
-    group: root
-    mode: 0644
-  with_items:
-    - ceph-1.3-rhel-7.conf
-    - comps-ceph-1.3-rhel-7.xml
-    - run-distill.conf
-    - variants-ceph-1.3-rhel-7.xml
+# This repository contains distill's configuration files.
+- name: clone rcm-metadata.git
```

Comment:
Yeah, I figured I would keep the git repo up to date outside of Ansible, because I really want Ansible to be as idempotent as possible. Same thing with using `state=latest` in a yum module... it can end up introducing changes to the system that I didn't explicitly intend to introduce.